test/util/artifacts/runas-memleak/README.md

```diff
@@ -48,42 +48,86 @@ instance of ExecuteShellCommand
 }
 
 Starting values for omiagent process:
-PID 33704
+PID 29574
 Thread count 3
-FD count 42
+FD count 40
 Memstats:
   PID   RSS    VSZ COMMAND
-33704  5740 302932 omiagent
-```
+29574  4608  16348 omiagent
```

Comment:
It's very odd that the RSS/VSZ here and after leak detection are identical. I think this output came from a run that as already stabilized, which makes it somewhat inaccurate.

Could you shut down the agent, rerun your script, and post the new numbers? I'd definitely expect growth from the starting values and the leak detection values.

Other than that, this looks fine. So please update and then go ahead and commit, thanks!